### ISSUE TITLE
Latest update breaks highlighting of variable declarations in JS/TS

### ISSUE THREAD

**Issue by andrewcourtice** [Monokai/monokai-pro-vscode#200]
Firstly, thanks for creating this great theme!

It appears as though the latest update to the theme has broken highlighting variable declarations in JavaScript/TypeScript. See screenshots below:

**v1.1.15**
<img width="480" alt="Screen Shot 2020-06-09 at 8 47 33 am" src="https://user-images.githubusercontent.com/11718453/84088238-6ac98d00-aa2f-11ea-9582-42bb91deeed3.png">

**v1.1.16** (latest version)
<img width="494" alt="Screen Shot 2020-06-09 at 8 46 45 am" src="https://user-images.githubusercontent.com/11718453/84088262-7452f500-aa2f-11ea-9d1e-18a4fc0321d1.png">

I have also tried toggling the `editor.semanticTokenColorCustomizations` VSCode setting but this appears to have no effect.

Any help would be greatly appreciated.

**Comment by reneweteling** [comment]
Same for the Elixir language

V1.1.16
![image](https://user-images.githubusercontent.com/3830933/84115569-913af880-aa2e-11ea-8dd5-cc9ade87802c.png)

v1.1.15
![image](https://user-images.githubusercontent.com/3830933/84115634-aa43a980-aa2e-11ea-87f8-154cecfd2000.png)

**Comment by reneweteling** [comment]
Amazing, just 1 hour later 1.1.17 is released and now it works again, thanks guys!

**Comment by Monokai** [comment]
I've reverted the change in 1.1.17.

The reasoning for the change in 1.1.16 was what I saw in Javascript code and especially what happened after toggling semantic highlighting on.

I like that variables defined in all caps are constants, they are meant to stand out and so it warrants a purple color. But, I also see that all the variable declarations like `const a = 0` are colored purple. I personally think that's too much. If it were up to me I would leave them just white, like `let a = 0` does, or how Sublime Text handles const variables. I understand const variables are supposed to be immutable, but it's too much purple for my taste. A const definition feels more lightweight to me than a variable declared in all caps for example.

But it's not always possible to treat all these cases differently, due to how the syntax definitions work in VSCode. I have to rely on them to color different parts of the code.

So, in the end I reverted the code to have the most optimal solution for all these cases, so it should be good again now.
